.vscode/settings.json

```diff
@@ -8,5 +8,17 @@
   "editor.defaultFormatter": "llvm-vs-code-extensions.vscode-clangd",
   "files.associations": {
     "*.inc": "cpp"
+  },
+  // CMake Configurations
```

Comment:
nit: comments are not permitted in JSON (flagged by my IDE)

Reply:
There is also a comment above, should we remove that one too?

https://github.com/EPFL-LAP/dynamatic/blob/9fef8a5929f4575206f39b48cb97b32e611bd8b8/.vscode/settings.json#L2

Reply:
Yes you can remove that one too, thanks for the catch :)

Reply:
Done!